src/blocks/block-coauthors/edit.js

```diff
@@ -0,0 +1,243 @@
+					className: classnames( {
+						[`is-layout-cap-${layout.type}`]: layout.type,
+						[`has-text-align-${ textAlign }`]: textAlign,
+					}),
+					style: {
+						gap: __experimentalGetGapCSSValue( attributes?.style?.spacing?.blockGap )
+					}
+				})
+			}>
+				{
+					coAuthors &&
+					'inline' === layout.type &&
+					( isSelected || prefix ) &&
+					(
+						<RichText
+							allowedFormats={ ALLOWED_FORMATS }
+							className="wp-block-cap-coauthors__prefix"
+							multiline={ false }
+							aria-label={ __( 'Prefix' ) }
+							placeholder={ __( 'Prefix' ) + ' ' }
+							value={ prefix }
+							onChange={ ( value ) =>
+								setAttributes( { prefix: value } )
+							}
+							tagName="span"
+						/>
+					)
+				}
+				{
+					coAuthors && 
+					coAuthors
+					.map( ( author ) => {
+						const isHidden = author.id === ( activeBlockContextId || coAuthors[0]?.id );
+						return (
+							<BlockContextProvider
+								key={ author.id }
+								value={ {'cap/author': author } }
+							>
+								{ isHidden ? (<CoAuthorTemplateInnerBlocks />) : null }
+								<MemoizedCoAuthorTemplateBlockPreview
+									blocks={blocks}
+									blockContextId={author.id}
+									setActiveBlockContextId={ setActiveBlockContextId }
+									isHidden={isHidden}
+								/>
+							</BlockContextProvider>
+						);
+					})
+					.reduce( ( previous, current, index, all ) => (
+						<>
+						{ previous }
+						{
+							'inline' === layout.type &&
+							(
+								<span className="wp-block-cap-coauthors__separator">
+									{ ( lastSeparator && index === (all.length - 1) ) ? `${lastSeparator}` : `${separator}` }
+								</span>
+							)	
+						}
+						{ current }
+						</>
+					))
+				}
+				{
+					coAuthors &&
+					'inline' === layout.type &&
+					( isSelected || suffix ) &&
+					(
+						<RichText
+							allowedFormats={ ALLOWED_FORMATS }
+							className="wp-block-cap-coauthors__suffix"
+							multiline={ false }
+							aria-label={ __( 'Suffix' ) }
+							placeholder={ __( 'Suffix' ) + ' ' }
+							value={ suffix }
+							onChange={ ( value ) =>
+								setAttributes( { suffix: value } )
+							}
+							tagName="span"
+						/>
+					)
+				}
+			</div>
+			<InspectorControls>
+				{
+					'inline' === layout.type &&
+					(
+						<PanelBody title={ __( 'CoAuthors Layout' ) }>
+						<TextControl
+							autoComplete="off"
+							label={ __( 'Separator' ) }
+							value={ separator || '' }
+							onChange={ ( nextValue ) => {
+								setAttributes( { separator: nextValue } );
+							} }
+							help={ __( 'Enter character(s) used to separate authors.' ) }
+						/>
+						<TextControl
+							autoComplete="off"
+							label={ __( 'Last Separator' ) }
+							value={ lastSeparator || '' }
+							onChange={ ( nextValue ) => {
+								setAttributes( { lastSeparator: nextValue } );
+							} }
+							help={ __( 'Enter character(s) used to distinguish the last author.' ) }
```

Comment:
```suggestion
							help={ __( 'Enter character(s) used to separate the last author.' ) }
```

Reply:
Addressed in https://github.com/Automattic/Co-Authors-Plus/pull/997/commits/52dc33dcae7a53317e61d48b679b292928f4c56b